src/layouts/core-dev-dot-layout/index.tsx

```diff
@@ -1,7 +1,14 @@
+import Head from 'next/head'
 import s from './core-dev-dot-layout.module.css'
 
 const CoreDevDotLayout: React.FC = ({ children }) => (
-  <div className={s.root}>{children}</div>
+  <>
+    <Head>
+      <link rel="icon" href="/favicon.ico" sizes="any" />
+      <link rel="icon" href="/icon.svg" type="image/svg+xml" />
```

Comment:
`.ico` is for Safari, which [doesn't seem to support `.svg`](https://caniuse.com/?search=svg%20favicon) (this is why on Safari I can't see a favicon for hashicorp.com, I think).

`.svg` is for other browsers, and like `hashicorp.com`, supports `prefers-color-scheme` for a dark mode icon (which can be seen in Firefox).